docs/source/getting-started/content/proc_basic-troubleshooting.adoc

```diff
@@ -60,7 +60,7 @@ If your issue has not been resolved by this procedure, perform the following ste
 . If no existing issue addresses the encountered issue, link:https://github.com/code-ready/crc/issues/new[create an issue] and link:https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests[attach the [filename]`~/.crc/crc.log` file] to the created issue.
 The [filename]`~/.crc/crc.log` file contains detailed debugging and troubleshooting information which can help diagnose the problem that you are experiencing.
 
-= DNS configuration details
+== DNS configuration details
 
```

Comment:
"editor"? Which editor?
I put it on purpose at the same level as "Basic Troubleshooting" and "Detailed Troubleshooting". If you want to move in inside "Detailed Troubleshooting", you need to change the "==" to "===" for "linux" and "mac" below I think.
But since there are already 2 sections starting with "=", I don't see why a 3rd one would trigger a warning?

Reply:
@cfergeau I am using goland with `adoc` plugin enabled and it was throwing that as warning.

Reply:
AFAIK, the heading levels should be absolutely fine as they were. We offset the levels when assembling this file into the master assembly, so essentially, all top-level headings here will end up as second-level headings. :)

Reply:
@robin-owen Ack I will remove this commit then and ignore the plugin warning :)